public/app/plugins/panel/bargauge/module.tsx

```diff
@@ -61,19 +61,41 @@ export const plugin = new PanelPlugin<Options>(BarGaugePanel)
         defaultValue: defaultOptions.showUnfilled,
         showIf: (options) => options.displayMode !== 'lcd',
       })
+      .addRadio({
+        path: 'sizing',
+        name: 'Bar size',
+        settings: {
+          options: [
+            { value: BarGaugeSizing.Auto, label: 'Auto' },
+            { value: BarGaugeSizing.Manual, label: 'Manual' },
+          ],
+        },
+        defaultValue: defaultOptions.sizing,
+        showIf: (options) => options.orientation !== VizOrientation.Auto,
```

Comment:
I think that this option should always be shown irregardless of the orientation (given auto orientation displays bars in either the horizontal or vertical layout, so all of the "manual" bar size options would apply in theory